app/controllers/project_content_items_controller.rb

```diff
@@ -21,13 +21,20 @@ def bulk_update
   end
 
   def flags
-    render :flags, locals: { project: project, content_item: content_item }
+    locals = { project: project, content_item: content_item }
+
+    respond_to do |format|
+      format.js { render layout: false, locals: locals }
```

Comment:
Do we still want to allow these flags to be updated without JS? With the `flags.html.erb` file removed, this'll only work with JavaScript. Perhaps we could include the flags partial (without the modal styling) so that we can provide both the JS enhancement and the HTML page.